tests/providers/aws/services/elbv2/elbv2_desync_mitigation_mode/elbv2_desync_mitigation_mode_test.py

```diff
@@ -119,7 +119,7 @@ def test_elbv2_without_desync_mitigation_mode_and_not_dropping_headers(self):
             assert len(result) == 1
             assert result[0].status == "FAIL"
             assert search(
-                "does not have desync mitigation mode set as defensive or strictest and is not dropping invalid header fields",
+                "does not have desync mitigation mode set as strictest and is not dropping invalid header fields",
```

Comment:
Please remove the `search` here. Don't you need to change the tests for them to pass?